centreon/tests/e2e/features/Cloud-notifications/05-notification-get.feature

```diff
@@ -9,20 +7,24 @@ Feature: List Notification Rules
     Given a user with access to the Notification Rules page
     And the user is on the Notification Rules page
 
-  Scenario: Empty notification rule list
-    When I have 0 rule
-    Then I see "No result found" on the table line
-    And pagination is disabled
+  Scenario: Empty Notification Rules List
+    Then I see "No result found" in the table
```

Comment:
When is required, even if it s a "no result case", for readability at least.
```suggestion
    Given no Notification Rules are configured
    When the user goes to Notification Rules Listing
    Then I see "No result found" in the table
```
Other point, what is "table", Scenario points to "List", this is confusing